manubot/pandoc/tests/test_bibliography.py

```diff
@@ -6,48 +6,47 @@
 from manubot.pandoc.bibliography import load_bibliography
 
 directory = pathlib.Path(__file__).parent
-bibliography_paths = sorted(
-    directory / x for x in directory.glob("bibliographies/bibliography.*")
+
+
+skipif_no_pdcp = pytest.mark.skipif(
+    not shutil.which("pandoc-citeproc"),
+    reason="pandoc-citeproc required to load this format.",
 )
-bibliography_path_ids = [path.name for path in bibliography_paths]
+skipif_no_pd = pytest.mark.skipif(
+    not shutil.which("pandoc-citeproc"), reason="pandoc required to load this format."
```

Comment:
What's this test condition?  Should it skip if pandoc is missing?

Reply:
yes! great catch. fixed in https://github.com/manubot/manubot/pull/256/commits/b1f7363b2f405262b8965b1a631b4e66ceaa0bd0